### ISSUE TITLE
Access to nextPageToken

### ISSUE THREAD

**Issue by demeero** [recurly/recurly-client-go#59]
Is it possible to get 'nextPagePath' value from your ***List entities?

I have a case when a client requests a plan list from my service and I request it from Recurly. And I would like to add pagination for the client, so he can pass 'nextPagePath' to my service and I propagate it to Recurly. But the problem is that I can't access 'nextPagePath' value in PlanList entity, because it is an unexported field. 

For example, Java SDK has public 'getNext()' method on Pager class, which returns nextPagePath and I can return it to my client.

**Comment by douglasmiller** [comment]
Hi @demeero,

We do not currently have any support for a resumable pager in the client libraries. The `getNext()` method that you're referencing in the Java client **will** provide you access to the cursor that points to the next page, but there is no mechanism available to restore a Pager with that cursor. In truth, that method (and some others) should have been declared as private methods in the Java client.

A resumable pager is a feature that we have on our todo list. I'll add a +1 for you.

**Comment by demeero** [comment]
Thanks for the answer.

I think it would have been possible to resume pager if 'nextPageToken' had been an exported field. 
Something like that:
```
        client := recurly.NewClient("api_key")
	planList := client.ListPlans(nil)
	if err := planList.Fetch(); err != nil {
		panic(err)
	}
	fmt.Printf("%v", planList.Data)
	nextToken := reflect.ValueOf(planList).Elem().FieldByName("nextTokenPath").String()

	// ....

	planList = recurly.NewPlanList(client, nextToken)
	if err := planList.Fetch(); err != nil {
		panic(err)
	}
	fmt.Printf("%v", planList.Data)
```

I don't test it, but maybe it works. But of course, nobody wants to use reflection to fetch a nextToken value.

Maybe I can somehow extend the SDK in my project to have a possibility to fetch a nextToken? Any suggestions?

**Comment by douglasmiller** [comment]
Hey @demeero, it's been a bit, but we just merged PR #80 which will allow access via the `Next()` method to access the data you wanted.
